font.go

```diff
@@ -118,10 +117,11 @@ var sysfontFinder = struct {
 func FindLocalFont(name string, style FontStyle) string {
 	// TODO: use style to match font
 	// try with fc-match first
-	filename, err := exec.Command("fc-match", "--format=%{file}", name).Output()
+	// TODO: this causes a panic on Linux "panic: requested font style not found"
```

Comment:
This optimization was causing problems on my end on Linux with svg's generated from graphviz. There was the panic shown here in the comments.

Reply:
The problem is because the fc-match is returning old X11 fonts on my system, which causes problems further down the line.

Reply:
I've added a custom system font finder with https://github.com/tdewolff/canvas/commit/b78baf2f085b2d9b690dbfe7d9baaf6b10d8e5d0, let me know how that goes.